physics/cires_ugwp_orolm97_v1.F90

```diff
@@ -0,0 +1,990 @@
+module cires_ugwp_orolm97_v1
+
+
+contains
+
+
+
+     subroutine gwdps_oro_v1(im,  km,    imx, do_tofd,           &
+         pdvdt, pdudt, pdtdt, pkdis, u1,v1,t1,q1,kpbl,           &  
+         prsi,del,prsl,prslk, zmeti, zmet, dtp, kdt, hprime,     &
+         oc, oa4, clx4, theta, sigmad, gammad, elvmaxd,  sgh30,  &
+         dusfc, dvsfc,  xlatd, sinlat, coslat, sparea,           &
+         cdmbgwd, me, master, rdxzb,                             &
+         zmtb, zogw, tau_mtb, tau_ogw, tau_tofd,                 &
+         dudt_mtb, dudt_ogw, dudt_tms)
+!----------------------------------------
+! ugwp_v1: gwdps_oro_v1 following recent updates of Lott & Miller 1997
+!          eventually will be replaced with more "advanced"LLWB
+!          and multi-wave solver that produce competitive FV3GFS-skills  
+! 
+!   computation of kref for ogw + coorde diagnostics
+!   all constants/parameters inside cires_ugwp_initialize.f90 
+!----------------------------------------    
+
+      use machine ,      only : kind_phys
+      use ugwp_common_v1,  only : rgrav,   grav,  cpd, rd, rv, rcpd, rcpd2, &
```

Comment:
Constants should come from the host model through the argument list, if possible.

Reply:
Done.